code/modules/recycling/disposal/holder.dm

```diff
@@ -87,6 +87,13 @@
 /obj/structure/disposalholder/proc/try_expel(datum/move_loop/source, succeed, visual_delay)
 	SIGNAL_HANDLER
 	if(current_pipe || !active)
+		if(!(locate(/obj/structure/disposaloutlet) in get_turf(src)))
+			for(var/mob/living/trashed_individual in src)
+				if(prob(10))
+					trashed_individual.apply_damage(1, BRUTE)
+					playsound(loc, 'sound/effects/wounds/crack2.ogg', 50, TRUE, FALSE)
+				if(prob(20))
```

Comment:
Might want to make this an `if else`, still keep your mutually exclusive 30% of a sound effect, but introducing the chance of hearing both with the number of chances players will have to hear it would be grating most likely, especially since they can still choose to manually clang the walls by moving.